src/main/java/Parser.java

```diff
@@ -1,21 +1,21 @@
 public class Parser {
-    public static Command parse(String fullcommand) throws DukeException {
-        int spaceIndex = fullcommand.indexOf(" ");
+    public static Command parse(String fullCommand) throws DukeException {
+        int spaceIndex = fullCommand.indexOf(" ");
```

Comment:
Can consider using regular expressions instead, as that well greatly ease the burden of parsing strings as commands here. See Java 11 Patterns.